### ISSUE TITLE
Doesn't play european games

### ISSUE THREAD

**Issue by fartboygh** [libretro/same_cdi#8]
At least not Atlantis - The Last Resort. It doesn't think it's a CDI disc.

**Comment by kiwisoup2016** [comment]
Yeah that's not right. That game not playing has nothing to do with it being European.
